packages/expo-image/src/ExpoImage.web.tsx

```diff
@@ -280,6 +31,19 @@ function onErrorAdapter(onError?: { (event: { error: string }): void }) {
   };
 }
 
+const SUPPORTED_ANIMATIONS = [
```

Comment:
Probably worth adding a type here
```suggestion
const SUPPORTED_ANIMATIONS: ImageTransition['effect'][] = [
```